test/eventlistener_test.go

```diff
@@ -302,21 +305,39 @@ func TestEventListenerCreate(t *testing.T) {
 	// ElPort forward sink pod for http request
 	portString := strconv.Itoa(*eventReconciler.ElPort)
 	podName := sinkPods.Items[0].Name
-	cmd := exec.Command("kubectl", "port-forward", podName, "-n", namespace, fmt.Sprintf("%s:%s", portString, portString))
-	err = cmd.Start()
+
+	config, err := clientcmd.BuildConfigFromFlags("", knativetest.Flags.Kubeconfig)
+	if err != nil {
+		panic(err)
```

Comment:
I just realized these "panic"s slipped through. I'll update them in the next patch.